src/app/users/users-update/users-update.component.html

```
@@ -138,46 +132,49 @@
     </div>
-  </form>
-</div>
-@if (!submissionMode) {
-  <div class="profile-image-column">
-    <div class="profile-image-content">
-      @if (showImagePreview) {
+    @if (!submissionMode) {
+    <div class="profile-image-column">
+      <div class="profile-image-content">
+        @if (showImagePreview) {
         <img class="profile-image-update" [src]="previewSrc">
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)